test/integration/navbar_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class NavbarTest < ActionDispatch::IntegrationTest
+	test "Valid links appear when nav buttons are clicked" do
+		get root_path
+        assert_template 'visualization/index'
+        assert_select "a[href=?]", visualization_index_path
+        assert_select "a[href=?]", visualization_create_path
```

Comment:
Remember to change the test to `visualization_new_path` too once it's changed from create to new in the navbar partial